NedyUdombat/Questioner
# [chore #163114814] Re-structure app architecture

#### What does this PR do?
This PR:
* Restructures the server folder

#### Description of Task to be competed:
* Rename all dummy folder to use actual names
* Rewrite all paths to use the new paths

#### How should this be manually tested?
* visit https://github.com/NedyUdombat/Questioner/
* and check ch-restructure-app-163114814 branch
* you will see and review the new structure

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories?(if applicable)
#163114814
